### PR TITLE
2 refine based on new release

### PR DESCRIPTION
**Pull request by ghost**
Added the changes for the new api.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,27 +1,79 @@
...
+                print(upload_key_json)
+                if 'jobId' in upload_key_json:
+                    _job_id = upload_key_json['jobId']
+                    # you can get the status of the job by polling or 
+                    # callback url defined in teh post request
+                    poll_status(job_id=_job_id)
+                else:
+                    print('missing job id key')
+        else:
+            print('dont know.')
```
Comment (0xSk3ttch) [inline main.py]: lol

File: main.py [hunk 2]
```diff
@@ -1,27 +1,79 @@
...
+        with open(f"./files/preamble.json", 'rb') as file:  # Path to file
+            _transcript = json.load(file)
+    except Exception as e:
+        raise e
+    
+    upload_key_endpoint = f"{API_URL}/redact"
+    # Call to get upload key
+    upload_key_json_body = {
+         #"entities": [] # you can leave the entities blank if you want to redact everything.
+        "entities": ["PERSON", "LOCATION"],  # Types of entities to be redacted
```
Comment (0xSk3ttch) [inline main.py:29]: Can you add an address where they can find the list of entity types? We should probably add that to a README here too.

File: main.py [hunk 3]
```diff
@@ -83,5 +136,5 @@ def poll_status(job_id = None):
 
 
 if __name__ == '__main__':
-    send_media()
-
+    #transcribe_and_redact()
```
Comment (0xSk3ttch) [inline main.py:140]: Remove parens

**Review by 0xSk3ttch — DISMISSED** [review]
I know this was my doing, but can you make sure we have consistency with single or double quotes?

File: main.py (hunk 3 above)
Comment (bluualpha) [reply main.py:139]: There are two different functions. One for transcription and one for redaction

**Comment by ghost** [comment]
Should be good with everything now.

**Pull request merged**